src/components/button/button.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Button } from 'react-native'
+import { ButtonColor } from '../../common/enums/enums'
+
+type Props = {
+  title: string
+  onClick?: (evt: React.MouseEvent<HTMLButtonElement>) => void
+  color?: ButtonColor
+}
+
+const CustomButton: React.FC<Props> = ({ title, onClick, color }) => {
+  return <Button title={title} onPress={onClick} color={color} />
```

Comment:
Button type (Primary, Secondary) should be here. We assume that it defines the styles of the button by the incoming type param